traefik.go

```diff
@@ -84,6 +84,28 @@ Complete documentation is available at https://traefik.io`,
 		},
 	}
 
+	//storeconfig Command init
```

Comment:
Note for later: we might want to split `traefik.go` a little bit 👼
(having a file for `storeconfig` command, same for `version`, …)